SIT.Manager/ViewModels/Play/ServerSummaryViewModel.cs

```diff
@@ -62,10 +62,27 @@ public string ShownAddress
         }
     }
 
+    public string ServerDisplayName
+    {
+        get
+        {
+            AkiServer? bookmarkedServer = _configService.Config.BookmarkedServers.FirstOrDefault(x => x.Address == _server.Address);
+            if (bookmarkedServer != null)
+            {
+                return !string.IsNullOrEmpty(bookmarkedServer.Nickname) ? bookmarkedServer.Nickname : Name;
+            }
+            return Name;
+        }
```

Comment:
We store a self reference to the server as _server which should be the same reference as what this code retrieves

Reply:
Fair point, I probably should have just sorted out the problem it caused in the first place 😄